superset/examples/configs/charts/Messages_per_Channel.yaml

```diff
@@ -50,7 +50,7 @@ params:
     dashboards: '#3CCCCB'
     design: '#1FA8C9'
     developers: '#9EE5E5'
-    embedd-dashboards: '#ACE1C4'
+    embedded-dashboards: '#ACE1C4'
```

Comment:
This is actually a value in the data. If we change this, it will break the example chart.